scripts/script_import_test.py

```diff
@@ -77,4 +77,8 @@ def mock_exists(unused_path):
                 'certifi==2021.5.30', '--target',
                 '../oppia_tools/certifi-2021.5.30',
                 '--user', '--prefix=', '--system',
+                sys.executable, '-m', 'pip', 'install',
+                'typing-extensions==4.0.1', '--target',
+                'third_party/python_libs',
+                '--user', '--prefix=', '--system',
```

Comment:
Why do we need this change?

Reply:
I used TypedDict in `core/feconf.py` and for this, we need the `typing-extensions` module. I added it to the requirements to install the module and included the corresponding test here.